src/main/java/org/checkerframework/checker/dependencyinjection/DependencyInjectionTransfer.java

```diff
@@ -41,8 +48,47 @@ public TransferResult<CFValue, CFStore> visitMethodInvocation(
               boundClassTypeMirror.getAnnotation(), diATF.classValValueElement, String.class);
 
       accumulate(node, result, classNames.toArray(new String[1]));
+    } else if (diATF.isAnnotatedWithMethod(node.getTree())) {
+      MethodInvocationNode methodInvocationNode = node;
+      MethodInvocationNode callToBindNode =
+          (MethodInvocationNode) methodInvocationNode.getTarget().getReceiver();
+      MethodInvocationNode namesClassNode =
+          (MethodInvocationNode) methodInvocationNode.getArgument(0);
+      FieldAccessNode boundClass = (FieldAccessNode) callToBindNode.getArgument(0);
+      StringLiteralNode givenName = (StringLiteralNode) namesClassNode.getArgument(0);
+
+      accumulateBindAnnotatedWith(
+          node, result, boundClass.getReceiver().getType().toString(), givenName.getValue());
     }
 
     return result;
   }
+
+  public void accumulateBindAnnotatedWith(
+      Node node, TransferResult<CFValue, CFStore> result, String value, String name) {
+    List<String> valuesAsList = Arrays.asList(value);
+    List<String> namesAsList = Arrays.asList(name);
+    // If dataflow has already recorded information about the target, fetch it and integrate
```

Comment:
This comment was copied from elsewhere (I think from `accumulate`) and is definitely not true of this code.